src/modules/FairFindPackage2.cmake.in

```diff
@@ -77,13 +134,21 @@ The ``ADD_REQUIREMENTS_OF`` argument is now a no-op and only remains for
 compatibility reasons. :command:`find_package2` now always merges all known
 requirements.
 
+.. versionadded:: 1.0.0
+
+By default :command:`find_package2` tries to determine the install prefix if
+the package is found. The result is populated to the variable
+:variable:`<pkgname>_PREFIX``. This behaviour can be disabled by setting the
+:variable:`FAIR_NO_AUTO_DETECT_PREFIX` to ``ON``.
+
 #]=======================================================================]
 
 macro(find_package2 qualifier pkgname)
   cmake_parse_arguments(ARGS "" "VERSION"
                         "COMPONENTS;ADD_REQUIREMENTS_OF;OPTIONAL_COMPONENTS" ${ARGN})
 
   string(TOUPPER ${pkgname} pkgname_upper)
+  string(TOLOWER ${pkgname} pkgname_lower)
```

Comment:
Should those two be `__…__` escaped and be `unset()` at the end?

Reply:
done, also changed the outvar argument to the `string()` function call as we discussed earlier.